### PR TITLE
[ticket/15097] Add PHP version to Board statistics

### PR DESCRIPTION
**Pull request by nomind60s**
The ACP main page shows the phpBB version and the database version, but didn't show the PHP version. With this change the PHP version is displayed as part of the Board statistics.

Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15097

### CODE REVIEW

**Comment by DavidIQ** [comment]
You're targeting the wrong branch (master). Needs to be the 3.1.x branch, although this probably should go into 3.2 instead.

**Comment by nomind60s** [comment]
I thought I targeted the 3.1.x branch. What did I do incorrect?

**Comment by DavidIQ** [comment]
When you created the pull request you needed to change the target branch, which you should still be able to do.

**Comment by nomind60s** [comment]
I believe all my work is done here. First phpBB bugfix.

**Comment by nomind60s** [comment]
Oops, wrong button. Sorry.

**Comment by nomind60s** [comment]
Closing and reopening to cause travis checks to re-run. They had previously passed.

**Comment by nomind60s** [comment]
Reopening to cause travis-ci checks to repeat.

**Comment by marc1706** [comment]
We can also restart if there are issues (which we usually do before merging).

**Comment by Lady-G** [comment]
There is a dependency on the language pack. There are two new variables not contained in my default American English language pack. The ACP displays the variables literally instead of the text description. I've updated the bug tracker with my comments.

**Comment by DavidIQ** [comment]
As per coding guidelines, developers and code contributors only need to worry about the default language pack, which has been taken care of here. Other languages are the responsibility of the language pack maintainers. There is already a process in place for them to be notified of changes and it is up to them to monitor for these changes and implement those themselves.

File: phpBB/adm/style/acp_main.html [hunk 1]
```diff
@@ -94,15 +94,45 @@
...
 	<table class="table1 two-columns no-header" data-no-responsive-header="true">
 		<caption>{L_FORUM_STATS}</caption>
 		<col class="col1" /><col class="col2" /><col class="col1" /><col class="col2" />
 	<thead>
 	<tr>
-		<th>{L_STATISTIC}</th>
-		<th>{L_VALUE}</th>
-		<th>{L_STATISTIC}</th>
-		<th>{L_VALUE}</th>
+		<th style="width: 25%">{L_STATISTIC}</th>
```
Comment (CHItA) [inline phpBB/adm/style/acp_main.html]: This should be moved to some css file i think.

Comment (nomind60s) [inline phpBB/adm/style/acp_main.html]: In principle I would agree. However if you grep for style=\"width: in adm/style/*.html there are ~70 occurrences already, so it certainly seems this is not handled consistently. 

Will investigate resolution.

Comment (Nicofuma) [inline phpBB/adm/style/acp_main.html]: you are right, it's not really consistent and the design of this part of the ACP is a bit unique in the ACP.

I think we can merge it like that.

**Pull request merged**